Arrowgene.Ddon.Database/Files/Database/Script/migration_stamp_sqlite.sql

```diff
@@ -7,4 +7,6 @@ CREATE TABLE ddon_stamp_bonus
     CONSTRAINT fk_stamp_bonus_character_id FOREIGN KEY ("character_id") REFERENCES ddon_character ("character_id") ON DELETE CASCADE
 );
 
-INSERT INTO "ddon_stamp_bonus" SELECT "character_id", "1900-01-01 00:00:00", 0, 0 FROM "ddon_character"
\ No newline at end of file
+INSERT INTO "ddon_stamp_bonus" SELECT "character_id", "1900-01-01 00:00:00", 0, 0 FROM "ddon_character";
+
+insert or ignore into ddon_wallet_point (character_id, type, value) select character_id, 5, 0 from ddon_character;
```

Comment:
insert or ignore is sqlite specific, you'd have to write something dbms agnostic like (untested)
```sql
insert into ddon_wallet_point(character_id, type, value)
select character_id, 5, 0
from ddon_character
where not exists(select * from ddon_wallet_point where ddon_wallet_point.character_id=ddon_character.character_id and type=5);
```